ctf/docs/design-audit-2026-05-19.md

```
@@ -0,0 +1,126 @@
+# Design Audit: Replit Mockup vs Shell Code
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)